.github/workflows/master-change.yml

```diff
@@ -109,7 +108,7 @@ jobs:
           registry: https://registry.npmjs.org/
       - uses: JS-DevTools/npm-publish@v1
         with:
-          token: ${{ secrets.ACTIONS_ACCESS_KEY }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm not 100% confident on this working, but based on the repo config. I think we should be OK
![image](https://user-images.githubusercontent.com/1530951/227220729-90d5f421-2578-4a64-b05a-efdded414a19.png)